src/constants.h

```diff
@@ -101,15 +106,16 @@ enum class Algorithm {
   kRs256Algorithm = -257
 };
 
-// The reset command has a few sub commands with the following representations.
+// The ClientPin command has these sub commands.
 enum class PinSubCommand : uint8_t {
   kGetPinRetries = 0x01,
   kGetKeyAgreement = 0x02,
   kSetPin = 0x03,
   kChangePin = 0x04,
   kGetPinToken = 0x05,
   kGetPinUvAuthTokenUsingUvWithPermissions = 0x06,
-  kGetUvRetries = 0x07
+  kGetUvRetries = 0x07,
+  kGetPinUvAuthTokenUsingPinWithPermissions = 0x09,
```

Comment:
I don't know if we want to add a comment about 0x08.

Reply:
We want to! Added.